.prettierignore

```diff
@@ -4,4 +4,5 @@ templates
 README.md
 *.yml
 *.yaml
-CONTRIBUTION.md
\ No newline at end of file
+CONTRIBUTION.md
+Frontend/coverage
```

Comment:
Instead of ignoring the repo that is being formed, we can ignore the following
1. coverage
2. coverage.xml
3. .coverage

Also is it possible to shift tests in the root directory? Ideally, the path of test files should be -
./tests/frontend/

Reply:
Also, do add a single end line at the end of the file

Reply:
> Instead of ignoring the directory that is being formed, we can ignore the following files
> 
> 1. coverage
> 2. coverage.xml
> 3. .coverage
> 
> Also is it possible to shift tests in the root directory? Ideally, the path of test files should be -
> ./tests/frontend/

React isn't detecting any test files outside the `src/` folder. It's their rule to reside test files in `src/` folder. Looking if I find any tweak for this.